Modules/ and Templates/ are installed by devel/cmake-modules.

Not upstreamable.
--- CMakeLists.txt.orig
+++ CMakeLists.txt
@@ -576,7 +576,7 @@
 
 # Install script directories.
 install(
-  DIRECTORY Help Modules Templates
+  DIRECTORY Help
   DESTINATION ${CMAKE_DATA_DIR}
   FILE_PERMISSIONS OWNER_READ OWNER_WRITE GROUP_READ WORLD_READ
   DIRECTORY_PERMISSIONS OWNER_READ OWNER_EXECUTE OWNER_WRITE
